<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>foodinordercard</class>
 <widget class="QFrame" name="foodinordercard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>150</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="FoodInOrderCardImageButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="FoodInOrderCardTextButton" name="pushButton_TextToFood">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="FoodNumber">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>数量</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="PricePer">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>金额</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="address">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>地址</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FoodInOrderCardImageButton</class>
   <extends>QPushButton</extends>
   <header>foodinordercardimagebutton.h</header>
  </customwidget>
  <customwidget>
   <class>FoodInOrderCardTextButton</class>
   <extends>QPushButton</extends>
   <header>foodinordercardtextbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_TextToFood</sender>
   <signal>clicked()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
